shared/oae/css/oae.base.css

```diff
@@ -897,3 +897,31 @@ ul.as-list li.as-result-item.active {
 .oae-markdown-embedded p:first-child {
     margin-top: 0;
 }
+
+/**
+ * Markdown table styles
```

Comment:
- I'm not convinced that the font size of tables should be larger than the regular font size, so let's remove that
- The left and right padding on regular table cells should be the same as those on table header cells. Otherwise they don't align
- We try not to use `bolder`. We use either `bold` or 500/600/700
- We probably do want to give the odd rows an explicit white background color. Currently it looks a bit odd in the activity feed because of the grey background.